src/include/CMakeLists.txt

```diff
@@ -170,6 +170,7 @@ install (FILES kdbextension.h
 	       "${CMAKE_CURRENT_BINARY_DIR}/kdbtypes.h"
 	       kdbhelper.h
 	       "${CMAKE_CURRENT_BINARY_DIR}/kdb.h"
+	       "${CMAKE_CURRENT_BINARY_DIR}/kdberrors.h"
```

Comment:
`kdberrors.h` should not be installed, it shouldn't be used outside of Elektra. Why did you add it?

Reply:
I removed it. I was not aware that this should not be installed

Reply:
@kodebach are you sure about that? The file might be helpful for out-of-source-tree development of plugins. As this is currently not a priority, I think it is a good thing to not install any private files.

If we install private files, we should install in a subdirectory (private).

Reply:
The old version of `kdberrors.h` wasn't suitable for installation IMO. The new version could be installed, but we should first decide, whether or not we want to remove the `F` variants of the macros (see https://github.com/ElektraInitiative/libelektra/issues/2490#issuecomment-531040664).

Reply:
I'll see how the new header file looks like.